.changes/unreleased/alloy-TINY-11549-2024-12-04.yaml

```diff
@@ -0,0 +1,6 @@
+project: alloy
+kind: Improved
+body: Centering an element now also considers boundaries.
```

Comment:
So this is no longer a change to alloy right? So remove this? We also need a changelog for the `tinymce` project since this is new behaviour.

Reply:
done